src/code/z_viscvg.c

```diff
@@ -1,70 +1,100 @@
-#include "global.h"
+/**
+ * @file z_viscvg.c
+ * Description: Visualise Coverage (how much of the neighbourhood of a pixel is covered by a primitive) in various ways.
+ *
+ * To understand this file, it is helpful to remember that A_MEM is essentially synonymous with coverage, and that
+ * `GBL_c1/2(p, a, m, b)` are usually `(p * a + m * b) / (a + b)`.
+ *
+ * Coverage is maximum when not on an edge, on an edge it is smaller, and since coverage is treated as an alpha edges
+ * show up as darker in all of the available modes.
+ *
+ * We abbreviate coverage to "cvg"; "pixel RGB" is the original colour the pixel had before the filter is applied.
+ */
 
-// Note : This file is related to z_vismono, the original name was probably z_vis<something before "mono"
-// alphabetically>
+#include "global.h"
 
-Gfx D_8012AC00[] = {
+/**
+ * Draws only coverage: does not retain any of the original pixel RGB, primColor is used as background colour.
+ */
+Gfx sCoverageOnlyDL[] = {
     gsDPSetOtherMode(G_AD_PATTERN | G_CD_MAGICSQ | G_CK_NONE | G_TC_CONV | G_TF_POINT | G_TT_NONE | G_TL_TILE |
                          G_TD_CLAMP | G_TP_NONE | G_CYC_1CYCLE | G_PM_NPRIMITIVE,
                      G_AC_NONE | G_ZS_PRIM | G_RM_VISCVG | G_RM_VISCVG2),
+    // (blendColor) * (cvg)
```

Comment:
I don't think we should start commenting on every blender/combiner usage, if anywhere this could go in gbi.h near the rendermode (note tharo is reformatting it so I wouldn't touch it for now for conflict reasons) (also we need to talk about what we can do to gbi.h comments-wise but that's completely unrelated to the PR)

Reply:
In general I'd agree, I'd regard this as an exception since here it's pretty significant to what the functions in this file do.